### PR TITLE
Finished Sending the Request to the Server From the form after clicking the button

### PR DESCRIPTION
**Pull request by irumvanselme**

### CODE REVIEW

File: src/com/customify/cli/components/Button.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.customify.client.components;
+
+public class Button {
```
Comment (makuzaverite) [inline src/com/customify/cli/components/Button.java:3]: I'm not sure why to added an empty class

**Review by makuzaverite — CHANGES_REQUESTED** [review]
It would be great if you could add the screen shots of the stuffs you've added, that would be awesome

File: src/com/customify/cli/components/Button.java (hunk 1 above)
Comment (irumvanselme) [reply src/com/customify/cli/components/Button.java:3]: ### Here are some screenshots
1. In the Code editor
image
2. Here is the response in the database
image

**Review by makuzaverite — CHANGES_REQUESTED** [review]
Thank you so much

**Review by makuzaverite — APPROVED** [review]
Thank you so much

**Pull request merged**